zap/src/main/java/org/zaproxy/zap/extension/brk/AbstractBreakPointMessage.java

```diff
@@ -19,6 +19,10 @@
  */
 package org.zaproxy.zap.extension.brk;
 
+/**
+ * @deprecated (2.15.0) See the break add-on in zap-extensions instead.
+ */
+@Deprecated
```

Comment:
Add the `since` and `forRemoval` to the annotation.

Reply:
can you type an example?

Reply:
https://github.com/zaproxy/zaproxy/blob/6ab053214ac636e9ccf89ee8b7004037f68b8484/zap/src/main/java/org/zaproxy/zap/extension/ascan/ActiveScript.java#L25

Reply:
thank you